clients/cobol-lsp-vscode-extension/package.json

```diff
@@ -150,7 +150,7 @@
         "watch": "tsc -watch -p ./",
         "postinstall": "node ./node_modules/vscode/bin/install",
         "test": "jest -w 1 --unhandled-rejections=strict",
-        "coverage": "jest --coverage"
+        "coverage": "jest --coverage --runInBand"
```

Comment:
run coverage in banc mode instead of running in parallel. Planning it to keep this flag for some time and observe the performance and the intermittent failing UI test.